packages/model/addon/-private/system/model-for-mixin.ts

```diff
@@ -3,15 +3,18 @@ import { getOwner } from '@ember/application';
 import Model from '../model';
 
 type Store = import('@ember-data/store').default;
-
-/*
+/* 
     In case someone defined a relationship to a mixin, for example:
     ```
-      let Comment = Model.extend({
-        owner: belongsTo('commentable'. { polymorphic: true })
-      });
-      let Commentable = Ember.Mixin.create({
-        comments: hasMany('comment')
+      import Model, { belongsTo, hasMany } from '@ember-data/model';
+      import Mixin from '@ember/object/mixin';
+
+      let Comment = class CommentModel extends Model {
```

Comment:
```suggestion
      class CommentModel extends Model {
```

Reply:
I honestly am not sure the intention of this Mixin block and what it is trying to communicate

Reply:
fixed